### PR TITLE
fix forgotten verticalpodautoscaler.Config

### PR DESCRIPTION
**Pull request by hervenicol**
@QuentinBisson 
Some fixes for your PMO branch.

### CODE REVIEW

File: service/controller/resource/resource.go [hunk 1]
```diff
@@ -267,9 +267,11 @@ func New(config Config) ([]resource.Interface, error) {
 	var verticalPodAutoScalerResource resource.Interface
 	{
 		c := verticalpodautoscaler.Config{
-			Logger:    config.Logger,
-			K8sClient: config.K8sClient,
-			VpaClient: config.VpaClient,
+			Logger:       config.Logger,
```
Comment (QuentinBisson) [inline service/controller/resource/resource.go:270]: I think you need to add that to the other resource.go file under the managementcluster controller

Comment (QuentinBisson) [reply service/controller/resource/resource.go:270]: https://github.com/giantswarm/prometheus-meta-operator/blob/6c67a80180f2833f972b78897028e56752bf3ffb/service/controller/managementcluster/resource.go#L200

Comment (hervenicol) [reply service/controller/resource/resource.go:270]: You mean `service/controller/managementcluster/resource.go` ? That you already did! :grin: But you only did one on them, so I also added it there.

Comment (QuentinBisson) [reply service/controller/resource/resource.go:270]: Ah awesome then

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**